docs/components/tabs.md

```diff
@@ -0,0 +1,278 @@
+---
+sidebarDepth: 0
+---
+# Tabs
+
+**KTabs** - A mindblowing tabs component
+<ClientOnly>
+  <KTabs :tabs="tabs">
+    <template v-slot:tab1>
+      <p>Tab 1 content</p>
+    </template>
+    <template v-slot:tab2>
+      <p>Tab 2 content</p>
+    </template>
+  </KTabs>
+</ClientOnly>
+
+```vue
+<KTabs :tabs="tabs">
+  <template v-slot:tab1>
+    <p>Tab 1 content</p>
+  </template>
+  <template v-slot:tab2>
+    <p>Tab 2 content</p>
+  </template>
+</KTabs>
+
+<script>
+export default {
+  data () {
+    return {
+      tabs: [
+        {
+          hash: '#tab1',
+          title: 'TAB 1'
+        },
+        {
+          hash: '#tab2',
+          title: 'TAB 2'
+        }
+      ]
+    }
+  }
+}
+</script>
+```
+
+## Props
+### tabs
+KTabs takes one required prop which is an array of tab objects.
+
+- `tabs`
+- `@change` - Emitted when a tab is changed
+
+```vue
+<template>
+  <KTabs :tabs="tabs" />
+</template>
+<script>
+export default {
+  data () {
+    return {
+      tabs: [
+        { hash: '#pictures', title: 'Pictures' },
+        { hash: '#movies', title: 'Movies' },
+        { hash: '#books', title: 'Books' },
+      ]
+    }
+  }
+}
+</script>
+```
+
+### default-tab
+By default the tabs will set the first tab in the array as active. You can override this by passing in the hash of any other tab to be used instead with v-model.
+<ClientOnly>
+  <KTabs v-model="defaultTab" :tabs="tabs">
+    <template v-slot:tab1>
+      <p>Tab 1 content</p>
+    </template>
+    <template v-slot:tab2>
+      <p>Tab 2 content</p>
+    </template>
+  </KTabs>
+</ClientOnly>
+
+```vue
+<KTabs
+  v-model="#tab2"
+  :tabs="tabs">
+  <template v-slot:tab1>
+    <p>Tab 1 content</p>
+  </template>
+  <template v-slot:tab2>
+    <p>Tab 2 content</p>
+  </template>
+</KTabs>
+```
+
+## Slots
+In order to actually see your tabbed content you must slot it using the tab hash property without the hash mark.
+
+<ClientOnly>
+  <KTabs :tabs="slottedTabs">
+    <template v-slot:pictures>
+      <p>Wow look Pictures!</p>
+    </template>
+    <template v-slot:movies>
+      <p>Wow look Movies!</p>
+    </template>
+    <template v-slot:books>
+      <p>Wow look Books!</p>
+    </template>
+  </KTabs>
+</ClientOnly>
+
+```vue
+<template>
+  <KTabs :tabs="tabs">
+    <template v-slot:pictures>Wow look Pictures!</template>
+    <template v-slot:movies>Wow look Movies!</template>
+    <template v-slot:books>Wow look Books!</template>
+  </KTabs>
+</template>
+<script>
+export default {
+  data ) {
+    return {
+      tabs: [
+        { hash: '#pictures', title: 'Pictures' },
+        { hash: '#movies', title: 'Movies' },
+        { hash: '#books', title: 'Books' },
+      ]
+    }
+  }
+}
+</script>
+```
+
+## Usage
+### Window Hash
```

Comment:
the example doesn't use window.location.hash anymore so might call this "Vue Router" and you could mention `window.location.hash` is also available. I don't think this is working btw. when I click a tab, the hash changes, but not the highlighted tab.

Reply:
updated to be more generic. Also seems to be working for me
![image](https://user-images.githubusercontent.com/5941111/71114070-d408cb80-2183-11ea-8825-9c45f7a57544.png)

Reply:
for the hash demo to reproduce the issue: click "Books", refresh the page, then click "Pictures"